README.md

```
@@ -1,32 +1,406 @@
+| `make init`   | Initialize development environment |
+| `make verify` | Run all checks and tests           |
+| `make test`   | Run unit and e2e tests             |
+| `make unit`   | Run unit tests only                |
+| `make e2e`    | Run e2e tests only                 |
+| `make check`  | Run lints and type checks          |
+| `make format` | Format code                        |
+| `make clean`  | Clean build artifacts              |
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)